bot/exts/info/code_snippets.py

```diff
@@ -228,8 +226,16 @@ async def on_message(self, message: Message) -> None:
 
             for pattern, handler in self.pattern_handlers:
                 for match in pattern.finditer(message.content):
-                    snippet = await handler(**match.groupdict())
-                    all_snippets.append((match.start(), snippet))
+                    try:
+                        snippet = await handler(**match.groupdict())
+                        all_snippets.append((match.start(), snippet))
+                    except ClientResponseError as error:
+                        error_message = error.message  # noqa: B306
```

Comment:
Out of interest, what was the reason for using `error.message` here with a noqa, instead of `str(error)`?

Reply:
Because it wouldn't consist solely of the message

https://github.com/aio-libs/aiohttp/blob/v3.7.4/aiohttp/client_exceptions.py#L87-L92

It's on a separate line cause the noqa wasn't working when it was in the f-string for some reason.